cpg-core/src/integrationTest/kotlin/de/fraunhofer/aisec/cpg/IntegrationTest.kt

```diff
@@ -23,35 +23,39 @@
  *                    \______/ \__|       \______/
  *
  */
-package de.fraunhofer.aisec.neo4j
+package de.fraunhofer.aisec.cpg
 
-import de.fraunhofer.aisec.cpg.TranslationManager
-import de.fraunhofer.aisec.cpg.TranslationResult
 import de.fraunhofer.aisec.cpg.graph.declarations.FunctionDeclaration
 import de.fraunhofer.aisec.cpg.graph.functions
 import de.fraunhofer.aisec.cpg.graph.statements.expressions.CallExpression
-import de.fraunhofer.aisec.cpg_vis_neo4j.Application
+import de.fraunhofer.aisec.cpg.persistence.createJsonGraph
+import de.fraunhofer.aisec.cpg.persistence.persistJson
 import java.nio.file.Paths
 import kotlin.io.path.createTempFile
-import kotlin.reflect.jvm.javaField
 import kotlin.test.assertEquals
 import kotlin.test.assertNotNull
 import org.junit.jupiter.api.Test
-import org.neo4j.ogm.annotation.Relationship
-import picocli.CommandLine
 
-fun createTranslationResult(file: String = "client.cpp"): Pair<Application, TranslationResult> {
+fun createTranslationResult(file: String = "client.cpp"): TranslationResult {
```

Comment:
No being in the `cpg-core` we probably cannot access the cxx language frontend anymore ^^
Is there an alternative way to create a graph that we can export for the test-case?